apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-styling.component.html

```diff
@@ -0,0 +1,35 @@
+<form [formGroup]="customForm">
+    <fieldset fd-fieldset>
+        <h2>Compact Checkbox with Reactive form and Error Status Style</h2>
+        <span>Checkbox with Error status</span>
+        <fdp-checkbox
+            [id]="'checkbox-10'"
+            [name]="'checkbox-10'"
+            [label]="'Mandatory checkbox'"
+            [binary]="true"
+            [size]="'compact'"
+            formControlName="example1"
+        ></fdp-checkbox>
+    </fieldset>
+</form>
+<div>Value: {{ customForm.get('example1').value }}</div>
```

Comment:
Please avoid using methods inside templates because it leads to performance issues. (methods being called every angular check cycle)

Reply:
replaced with customForm.getRawValue() | json.